sample-app/src/pages/VerticalSearchPage.tsx

```diff
@@ -54,6 +55,17 @@ const staticFiltersGroupLabels = {
   c_employeeCountry: 'Employee Country'
 }
 
+const filterSearchFields = [{
+  fieldApiName: 'builtin.location',
+  entityType: 'ce_person',
+  fetchEntities: false
```

Comment:
If we enable `fetchEntities`, it won't affect the FilterSearch component at all, correct? Because of that, I'm thinking we shouldn't have `fetchEntities` as an option for the component. We could use a Typescript utility type to omit the field, and inside the component we can always set it to false

Reply:
Updated!